test/import/FSHImporter.test.ts

```diff
@@ -84,6 +84,33 @@ describe('FSHImporter', () => {
     assertAssignmentRule(profile.rules[2], 'bonusCode', expectedBonusCode);
   });
 
+  it('should parse unicode characters in strings and multiline strings', () => {
+    const input = leftAlign(`
+    Profile: Escape
+    Parent: Observation
+    Title: "Just going to \\u270E in some unicode"
+    * valueString = "This cool \\u2603 can read music in \\uD834\\uDD1E!"
+    * category.text = """
+      When there is a sign of \\u2744, often school is canceled.
+      Kids can stay home and play with \\uD83C\\uDC31 or do anything!
+    """
+    `);
+    const result = importSingleText(input, 'Escape.fsh');
+    const profile = result.profiles.get('Escape');
+    const expectedCategoryText = [
+      'When there is a sign of \u2744, often school is canceled.',
+      'Kids can stay home and play with \uD83C\uDC31 or do anything!'
+    ].join('\n');
+    expect(profile.title).toEqual('Just going to \u270E in some unicode');
+    expect(profile.rules).toHaveLength(2);
+    assertAssignmentRule(
+      profile.rules[0],
+      'valueString',
+      'This cool \u2603 can read music in \uD834\uDD1E!'
+    );
+    assertAssignmentRule(profile.rules[1], 'category.text', expectedCategoryText);
+  });
```

Comment:
I think you should add another test, but in this one use the real unicode characters.  E.g.:
```ts
    const input = leftAlign(`
    Profile: Escape
    Parent: Observation
    Title: "Just going to ✎ in some unicode"
    * valueString = "This cool ☃ can read music in 𝄞!"
    * category.text = """
      When there is a sign of ❄, often school is canceled.
      Kids can stay home and play with 🀱 or do anything!
    """
    `);
```

We want to make sure that works too. (I tested it manually and it seems to work.)

Reply:
Added in [a490e74](https://github.com/FHIR/sushi/pull/1233/commits/a490e740b9eec3d9c12ea61cf51de7cc44a21950).